test/unit/reductions/exponentiated_gradient/test_exponentiatedgradient_smoke.py

```diff
@@ -98,9 +105,9 @@ def Q(X): return expgrad._pmf_predict(X)[:, 1]
     def test_smoke(self, testdata):
         self.run_smoke_test(testdata)
 
-    def test_simple_fit_predict(self):
+    @pytest.mark.parametrize("constraints", constraints)
```

Comment:
nit: I'd prefer using the list of class names
```
@pytest.mark.parametrize("Constraints", [DemographicParity, EqualizedOdds, ...])
```
and then instantiate it inside the test using
```
constraints = Constraints()
```
because we may add more complex logic to those constraint objects at some point. Better to have a fresh one for each test as opposed to reusing them which is what you're doing with the `constraints` list here.

Reply:
Done 👍